src/main/domain/cart.go

```diff
@@ -0,0 +1,6 @@
+package domain
+
+type Cart struct {
+	ID    string `json:"id"`
+	Category  string `json:"category,omitempty"`
```

Comment:
Esta muy chevere ver que estas estructurando el proyecto! Se recomienda que el [nombre de los paquetes significativo](https://golang.org/doc/effective_go.html#package-names) (meaningful), esto implica no usar nombres ["malos" nombres](https://blog.golang.org/package-names)

"Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains."

Podemos revisarlo en la sesión de P&R de hoy.